src/data/faq_de.json

```diff
@@ -1158,12 +1158,14 @@
                                 "anchor": "vac_booster_jj",
                                 "active": false,
                                 "textblock": [
-                                    "<b>AKTUELL 2. Februar 2022</b><br/>Siehe auch diesen FAQ Artikel bezüglich <a href='#vc_2of1' target='_blank'>2/1-Impfzertifikaten</a>.",
-                                    "<b>AKTUELL 30. Dezember 2021</b> <br/> Kürzlich hat die Europäische Kommission die Vorschriften für die Kodierung von Impfzertifikaten angepasst. Dies wird in Zukunft ermöglichen, dass Impfzertifikate, die den Abschluss der ersten Impfserie belegen, immer von Impfzertifikaten unterschieden werden können, die nach einer <a href='#glossary_auffrischimpfung' target='_blank'>Auffrischimpfung</a> ausgestellt wurden. <a href='#glossary_auffrischimpfung' target='_blank'>Auffrischimpfungen</a> werden in Zukunft wie folgt gekennzeichnet:",
+                                    "<b>Aktualisiert am 2. Februar 2022</b>",
+                                    "<div class='update'>Siehe auch diesen FAQ Artikel bezüglich <a href='#vc_2of1' target='_blank'>2/1-Impfzertifikaten</a>.</div>",
+                                    "<b>Aktualisiert am 30. Dezember 2021</b>",
+                                    "<div class='update'Kürzlich hat die Europäische Kommission die Vorschriften für die Kodierung von Impfzertifikaten angepasst. Dies wird in Zukunft ermöglichen, dass Impfzertifikate, die den Abschluss der ersten Impfserie belegen, immer von Impfzertifikaten unterschieden werden können, die nach einer <a href='#glossary_auffrischimpfung' target='_blank'>Auffrischimpfung</a> ausgestellt wurden. <a href='#glossary_auffrischimpfung' target='_blank'>Auffrischimpfungen</a> werden in Zukunft wie folgt gekennzeichnet:",

```

Comment:
`<div class='update'`

The tag is incomplete and not closed.

It should be `<div class='update'>`